modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/product_options/ProductOptionText.js

```diff
@@ -58,7 +58,7 @@ const ProductOptionText = ({
 			);
 
 			if (option) {
-				[value] = option.value;
+				value = option.value;
```

Comment:
@alessiorendina could you please have a look to this you too? Because I don't know the reason why you used arrays at the beginning

Reply:
This is for the minicart edit options. Basically the value coming from the json of the order item is always an array.

Reply:
Please @brianikim revert those kind of changes